components/automate-ui/src/app/modules/desktop/desktop-detail/desktop-detail.component.scss

```diff
@@ -122,6 +122,24 @@ h3 {
       display: none;
     }
 
+    span:last-child {
+      display: block;
+    }
+
+    chef-icon {
+      &.converged, &.unchanged {
+        color: #4FC293;
+      }
+
+      &.missing, &.unknown {
+        color: #7C8296;
+      }
+
+      &.failure, &.error {
+        color: #EB372C;
+      }
```

Comment:
Consider making these named constants from _variables.scss

Reply:
@msorens this is on the detail view, so after loading data and goign to the dashboard desktop view, select an error to see the list of machines, then select an item from that list to see the detail view